render/render.js

```diff
@@ -891,11 +936,26 @@ module.exports = function($window) {
 		if (dom.vnodes == null) dom.textContent = ""
 
 		vnodes = Vnode.normalizeChildren(Array.isArray(vnodes) ? vnodes : [vnodes])
+		var prevHookErrors = hookErrors
+		var prevHookThrown = hookThrown
+		hookThrown = false
+		hookErrors = []
 		updateNodes(dom, dom.vnodes, vnodes, hooks, null, namespace === "http://www.w3.org/1999/xhtml" ? undefined : namespace)
 		dom.vnodes = vnodes
 		// `document.activeElement` can return null: https://html.spec.whatwg.org/multipage/interaction.html#dom-document-activeelement
 		if (active != null && activeElement() !== active && typeof active.focus === "function") active.focus()
 		for (var i = 0; i < hooks.length; i++) hooks[i]()
+		var nextHookThrown = hookThrown
+		var nextHookErrors = hookErrors
+		hookThrown = prevHookThrown
+		hookErrors = prevHookErrors
+		if (nextHookThrown) {
+			var message = "Errors occurred during rendering:"
+			for (var i = 0; i < nextHookErrors.length; i++) message += "\n" + nextHookErrors[i].message
```

Comment:
Shouldn't we rather use `console.error(nextHookErrors[i])` so that the stack is available and interactive in the console, then throw `"Errors occurred during rendering"`?